itests/src/test/java/org/apache/unomi/itests/PropertiesUpdateActionIT.java

```diff
@@ -71,6 +71,7 @@ public void setUp() throws IOException, InterruptedException {
         profile.setProperties(new HashMap<>());
         profile.setProperty("lastName", "Jose"); // property that have a propertyType registered in the system
         profile.setProperty("prop4", "New property 4"); // property that do not have a propertyType registered in the system
+        profile.setProperty("prop4", "New property 4"); // property that do not have a propertyType registered in the system
```

Comment:
This looks like a copy-paste problem ? The line is identical to the previous one ?

Reply:
yes you are right, I will remove this line.
Thx for the review !